website/docs/widgets/table.md

```diff
@@ -0,0 +1,155 @@
+# Table
+
+MiraiTable allows you to build the Flutter Table widget using JSON.
+To know more about the Table widget in Flutter, refer to the [official documentation](https://api.flutter.dev/flutter/widgets/Table-class.html).
+
+## Properties
+
+| Property                 | Type                                | Description                                                                                                                                                                     |
+| ------------------------ | ----------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| columnWidths             | `Map<String, Map<String, dynamic>>` | Determines the column width on a column by column basis, if a particular column entry has null width `defaultColumnWidth` is used.                                              |
```

Comment:
The type should be `Map<int, MiraiTableColumnWidth>?`. Also please add docs for `MiraiTableColumnWidth`